metadata/src/main/java/edu/unc/lib/dl/util/DepositStatusFactory.java

```diff
@@ -209,19 +169,19 @@ public void fail(String depositUUID) {
 		fail(depositUUID, null);
 	}
 
-	/**
-	 * Delete deposit status.
-	 * @param depositUUID
-	 */
-	public void delete(String depositUUID) {
+	public void deleteField(String depositUUID, DepositField field) {
 		try (Jedis jedis = getJedisPool().getResource()) {
-			jedis.del(DEPOSIT_STATUS_PREFIX + depositUUID);
+			jedis.hdel(DEPOSIT_STATUS_PREFIX + depositUUID, field.name());
 		}
 	}
-
-	public void deleteField(String depositUUID, DepositField field) {
+	
+	/**	 
+	* Delete deposit status. For testing purposes only.
+	* @param depositUUID
+	*/
+	public void delete(String depositUUID) {
 		try (Jedis jedis = getJedisPool().getResource()) {
-			jedis.hdel(DEPOSIT_STATUS_PREFIX + depositUUID, field.name());
+			jedis.del(DEPOSIT_STATUS_PREFIX + depositUUID);
```

Comment:
I don't mind this sticking around, but I will mention that you don't really need to clean up your keys after yourself in the test since there should be basically no chance of a collision with the keys since you are generating new UUID's with each test.